test/system-test/exec.stest.ts

```diff
@@ -18,6 +18,7 @@ import { copySync } from 'fs-extra';
 import { homedir } from 'node:os';
 import { join } from 'path';
 import { cwd } from 'process';
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Please create follow up to fix it properly.

Reply:
its just a naming convention which should not apply on this, because we are importing classes which are named in capital letters

Reply:
Do we maintain it or classes belong to 3rd-party? Is it autogenerated classes or written by us?